.github/workflows/rust.yml

```diff
@@ -123,9 +115,9 @@ jobs:
         run: |
           export ARROW_TEST_DATA=$(pwd)/testing/data
           export PARQUET_TEST_DATA=$(pwd)/parquet-testing/data
+          cargo test
```

Comment:
We were previously only running `cargo test` in the `datafusion-examples` crate in this step